### DISCUSSION TITLE
Deserialize Allocates messages

### DISCUSSION THREAD

**Discussion by pattymack** [jamescourtney/FlatSharp#342] [Q&A]
Hello,
I'm looking for a way to NOT have Flatsharp allocate my message objects during deserialize.

So, instead of this API:
MsgClass msg = FlatBufferSerializer.Default.Parse<MsgClass>(buffer);

Would like to do my own allocation (pooling of objects), which would mean something like:
FlatBufferSerializer.Default.Parse<MsgClass>(buffer, ref MsgClass msg);

Is there a way to do this already?  I couldn't find it. 
Thanks

**Comment by jamescourtney** [comment]
No, not today. Please provide any feedback in #339 !
